db/migrate/20200329191558_create_users.rb

```diff
@@ -0,0 +1,15 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
```

Comment:
For all other records, we're using `id: :uuid`. Do you know if this is supported by Clearance?

Reply:
I had assumed, incorrectly it turns out, that the default had been changed to uuid and didn't need to be specified. Any objection to me changing the config so the migration generator default to `id: :uuid`?

Reply:
Hmm, it has been set https://github.com/jdbann/table/blob/master/config/application.rb#L21

Perhaps Clearance doesn't follow this?

Reply:
Fixed in b3d1ec5, yes Clearance copies the migration in and doesn't use the generator.